src/diagonal.works/b6/api/functions/graph.go

```diff
@@ -168,13 +168,15 @@ func (o *odCollection) Less(i, j int) bool {
 // Walking, with the default speed of 4.5km/h:
 // mode=walk
 // Walking, a speed of 3km/h:
-// mode=walk, walking speed=3.0
+// mode=walk, walk:speed=3.0
 // Transit at peak times:
 // mode=transit
 // Transit at off-peak times:
 // mode=transit, peak=no
 // Walking, accounting for elevation:
-// elevation=true (optional: uphill=hard downhill=hard)
+// elevation=true (optional: elevation:uphill=hard elevation:downhill=hard)
+// Walking, accounting for elevation, adding 12s per meter climbed:
+// elevation=true, elevation:naismith=12
```

Comment:
UMMMMMMMMM idts

what s the point of having a "rule" and then changing it / also no one knows who naismith is / it's meant to represent smth meaningful for a normal human being walking uphill doubling it makes no sense 

wdyt about exposing the uphill hard / downhill hard arbitrary coefficient? that s what it s there for actually / and that will have the same effect as this but i think makes more sense / it's the 1.2 thingy in line 205

Reply:
Fair. I've taken a similar approach to what you suggest.